lms/djangoapps/course_home_api/toggles.py

```diff
@@ -9,6 +9,15 @@
 
 WAFFLE_FLAG_NAMESPACE = LegacyWaffleFlagNamespace(name='course_home')
 
+# .. toggle_name: course_home.course_home_mfe
+# .. toggle_implementation: ExperimentWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: This experiement flag enables the use of the learning microfrontend (frontend-app-learning)
+#   as the default course frontend.
+# .. toggle_warnings: For this flag to have an effect, the LEARNING_MICROFRONTEND_URL setting must be configured, too.
+# .. toggle_use_cases: opt_in
```

Comment:
@edx/teaching-and-learning: Is the intent to remove the old frontend at some point? Should `toggle_use_cases` be `temporary`, and is there any target removal date, even if it is just a distant estimate?

Reply:
Good questions @robrap .

Engage (@jmyatt , @mikix , et al) is driving the cut-over of Course Home from the legacy frontend to the Learning MFE; I'll defer to them.

Reply:
I would assume that like the courseware toggles, we'd like to fully remove this course-home waffle flag by Lilac+1? So I'd say temporary with maybe a target date of that? But I'll double-defer to @jmyatt

Reply:
Should I put 09-10-2020 as this would be the current cut-off date for the maple.master branch?

Reply:
Sounds good to me

Reply:
sorry to have missed this.  It all looks good to me